runbooks/OrphanedVirtualMachineImages.md

```diff
@@ -0,0 +1,77 @@
+# OrphanedVirtualMachineImages
+
+## Meaning
+
+This alert fires when a VMI (virt-launcher-* pod) is running on a node that does not have a running `virt-handler-*` pod.  
+
+## Impact
+
+When a(n) VMI(s) is running on a node that does not have a running `virt-handler-*` those VMIs are considered orphaned. Meaning that they are no longer able to be managed.
+
+## Diagnosis
+
+You can confirm the alert by finding which nodes your virt-hanlder pods are running on.
+
+```bash
+kubectl -n kubevirt get pods -o wide -l kubevirt.io=virt-handler
+```
+*Output:*
+```
+NAME                 READY   STATUS    RESTARTS   AGE  IP               NODE     NOMINATED NODE   READINESS GATES
+virt-handler-vhqsp   1/1     Running   0          4h   10.244.140.80    node02   <none>           <none>
+virt-handler-xd8jc   1/1     Running   0          4h   10.244.196.168   node01   <none>           <none>
+```
+
+Then check to see which nodes the VMI(s) are running on. If they are running on a node that a virt-handler pod does not exist on those VMIs are orphaned.
```

Comment:
...not exist on those VMIs are orphaned --> not exist on - those VMIs are orphaned.
(need to add a separator to make the sentence more clear)